cmd/worker/main.go

```diff
@@ -295,7 +295,7 @@ func main() {
 	go cron.AutoImport(ctx, 15*time.Second, upstream)
 
 	// run container sync scheduler every minute
-	go cron.ContainerSync(ctx, 1*time.Minute, upstream)
+	go cron.ContainerSync(ctx, time.Duration(schedulerInterval)*time.Minute, upstream)
```

Comment:
I believe this should be the syncer interval
```suggestion
	go cron.ContainerSync(ctx, time.Duration(syncerInterval)*time.Minute, upstream)
```

Reply:
This process is the one updating the queue of jobs right ? 
schedulerInterval is in minute whereas syncerInterval is in seconds